corehq/apps/hqwebapp/templates/home.html

```diff
@@ -10,8 +10,8 @@
 {% endblock %}
 {% block head %}{{ block.super }}
     <link rel="stylesheet" href="{% static 'hqwebapp/css/mainlogin.css' %}"/>
-    <link rel="stylesheet" href="{% static 'hqwebapp/stylesheets/blueprint/screen.css' %}" media="screen, projection">
```

Comment:
I suspect this file isn't useful anymore - is there a use case for having prelogin disabled? Guessing we aren't keeping it up to date [("over 10 countries worldwide"!).](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/templates/home.html#L34)